src/Voucherify/DataModel/Gift.cs

```diff
@@ -18,9 +21,15 @@ public Gift WithAmount(int amount)
             return this;
         }
 
```

Comment:
Balance is not available for create/update operations, It's a value calculated on Voucherify side. We should create then separated class used when Voucher is created/updated to avoid mistakes in the future.